frontend/src/stores/theme.ts

```diff
@@ -0,0 +1,28 @@
+import { writable } from 'svelte/store';
+
+function getInitialTheme() {
+  const savedTheme = localStorage.getItem('theme');
+  if (savedTheme !== null) {
+    return savedTheme;
+  }
+  return window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
+}
+
+export const theme = writable<string>(getInitialTheme());
```

Comment:
For having stores synced to localStorage, there's already `localStorageSyncedStore`, can you use that?